sensors/Makefile

```diff
@@ -4,26 +4,49 @@
 # Copyright 2022 Phoenix Systems
 #
 
+LOCAL_PATH := $(call my-dir)
+
 # Sensors Server API Library
 NAME := libsensors
 include $(static-lib.mk)
 
+# FIXME: should be stored and compiled outside 
 # Sensors SPI interface
 NAME := libsensors-spi
 LOCAL_HEADERS := sensors-spi.h
-LOCAL_HEADERS_DIR := nothing  # do not reinstall libsensors.h
+LOCAL_HEADERS_DIR := nothing  # do not reinstall libsensors/*.h
 LOCAL_SRCS := libsensors-spi.c
 DEPS := libzynq7000-gpio-msg libspi-msg
 include $(static-lib.mk)
 
-# Sensors Server
-SENSORS_ALL := imu/lsm9dsxx imu/mpu6000 mag/lsm9dsxx mag/lis2mdl baro/lps25xx baro/ms5611 gps/pa6h gps/ubx
-SENSORS_SIM := imu/imu_sim.c mag/mag_sim.c baro/baro_sim.c gps/gps_sim.c gps/gps_sim.c
-SENSORS_LOCAL := $(SENSORS_SIM) $(addsuffix .c, $(filter $(SENSORS_ALL), $(DEVICE_SENSORS)))
+
+# Simulated sesnsors common library
+NAME := libsimsensors_common
```

Comment:
hmm, mayybe all of the common parts (simsensor common, gps common, etc.) should just be a part of `libsensors` - unused functions just will be removed while linking. Right now libsensors is a `dummy` lib providing only header files - so probably shouldn't be a library at all?

Reply:
Merged common sensor code into `libsensors` with clear include structure.